### PR TITLE
Equalizes Roundstart Mining Options (Removes some Advanced Scanners and Explorer Webbings)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
del: Removed Advanced Mining Scanner from roundstart voucher kits.
del: Removed Explorer Webbing from all voucher kits besides the shelter capsule pack.
/:cl:

Advanced Mining Scanner was removed to allow other packs to compete. When people are taking packs explicitly for the scanner, then it should probably be gated. If the lesser one is so trash then a separate PR can be made to remove the lesser and make the advanced be the default.

Explorer Webbing was removed for the very same reason, especially considering lockers come with shelter capsules basically making this choice a noob trap. Now you have to explicitly choose for the extra storage roundstart.

I've checked but if you are _filled with determination_ you can try to see if I overlooked anything by taking conscript for yourself lol.

This is, imo, a more sane way than #38123 which tried to arms race with other packs rather than just removing the offending items that made the select kits so desirable (that weren't related to the actual purpose of the kit, gotcha's get out).

Before I get the "resonators suck" flood, that's not my problem. It's not like they were being used anyways, people were just taking the kit for the desired items (hint: not the resonator) and just leaving it at base lol. This PR will have minimal effect on resonator **usage** (if such exists outside of golems). Hopefully someone can buff the poor thing.

Closes #38197

### CODE REVIEW

File: code/modules/mining/machine_vending.dm
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mining/machine_vending.dm]: Probably someone who doesn't into maps, as Meta comes with red toolboxes in mining but box does not.

**Comment by 81Denton** [comment]
>Hopefully someone can buff the poor thing.

Aside from the slow speed and melee range - it's incredibly clunky to use due to the 0.8s click delay between placing a field and being able to burst it. Maybe lowering it to CLICK_CD_RANGE (0.4s) would help?
Anywho, that's for another PR.

**Comment by Kromgar** [comment]
My only major issue with this is that no advanced scanners majorly impacts the mineral returns of first 10-15 minutes of a round until miners can buy their own. The scanner only got added to other packs to make them desireable by denton iirc.

I really could give 0 shits about the webbings. They are pretty cheap to buy if you absolutely need it. 

Although this makes the minebot and crusher the best starter packs for sure.

Ive grown to enjoy the crusher far more than kinetic accelerators

**Comment by 81Denton** [comment]
@ExcessiveUseOfCobblestone I'll test it once my apartment is less saunalike fug

**Comment by ExcessiveUseOfCobblestone** [comment]
@Kromgar if things are balanced around the advanced scanner being the default then let's look into making it a default post, otherwise I don't find that necessarily a bad thing that they have items to progress to that isn't boss items.

**Comment by Shadowflame909** [comment]
Eh. I feel like it wouldn't really effect mining either way if this gets merged. Just a slight timegate.

**Comment by Repukan** [comment]
How TF will the absence of advanced scanner affect mineral drop.

It still shows your whole screen. The fading effect lasts long enough to have less than a tick with nothing shown. I'd agree if you had some sort of advanced Alzheimer's but other than that it really doesn't affect much.

**Comment by SpaceManiac** [comment]
This makes `belt/mining/alt` inaccessible except for Legion drops.

>It still shows your whole screen.

The non-advanced mining scanner has a range of 4 to the advanced's 7.

**Comment by 81Denton** [comment]
It's a pretty big difference if you're running around and don't have to wait for the regular scanner to refresh. The added range is very nice for finding rarer stuff like diamonds too.
[Regular](https://user-images.githubusercontent.com/32391752/40685390-73bffffe-6394-11e8-863c-fc834c244be4.PNG) [Advanced](https://user-images.githubusercontent.com/32391752/40685392-7615f024-6394-11e8-8e41-7d8638077a6f.PNG)

**Comment by Bluespace** [comment]
Can we just have an advanced scanner kit?

**Comment by GuppyLaxx** [comment]
`But why?`

**Comment by ExcessiveUseOfCobblestone** [comment]
@SpaceManiac now it’s a super rare variant :-)

@GuppyLax not sure, have you checked the first post that requires you to explain the change and why?

**Review by Cyberboss — APPROVED** [review]

**Comment by Tlaltecuhtli** [comment]
how about just removing the resonator kit and idk put a jaunter a soap and call it space time slip kit

**Comment by Tlaltecuhtli** [comment]
make the adv scanner kit but its a cursed adv scanner that attracts bubblegum to its position

**Comment by zaracka** [comment]
The resonator kit is for species like golems, aliens, and whatnot. Keep it for when this happen for whatever reason. Wizards, events, etc.

**Comment by Tlaltecuhtli** [comment]
@zackara even if its golem related it shouldnt be here, the vouchers exist only on roundstart miners, free golems start with resonators and not vouchers

**Comment by 81Denton** [comment]
He probably meant if the wizard event changes you into a wizmeme before you have a chance to redeem your roundstart voucher

**Comment by zaracka** [comment]
In fact, how about making vouchers purchasable for an acceptable amount of points? There's lots of possibilities for being unable to use accelerators, not just being a free golem.

**Comment by Tlaltecuhtli** [comment]
what if vouchers were also a cargo crate and it was put instead of the current conscription crate which is a lesser of the conscription kit bought from the  vendor or from  voucher?

**Comment by ExcessiveUseOfCobblestone** [comment]
Buff reson 2020

**Comment by 81Denton** [comment]
@ExcessiveUseOfCobblestone I tested your PR now you have to do that, sperg's oath

**Comment by 81Denton** [comment]
(also nerf 9 range 40 brute dmg mech plasma cutters sshh)

**Comment by Kromgar** [comment]
@81Denton Mechs are awful on lavaland anyway. I doubt a 9 range plasma cutter is even capable of taking anything out beyond maybe ash drake and legion

**Comment by 81Denton** [comment]
@Kromgar Plasma cutter firefighter is the go to way to cheese megafauna (if you have no dignity), but I‘m dumb for bringing it up in an unrelated PR anywho

**Comment by SgtxRootbeer** [comment]
The firefighter requires you going to robotics tho

**Comment by Shadowflame909** [comment]
You can take out the hierophant with a literal mining borg. All you had to do is R U N. Once you master the art of that, sec tasers and hierophant is easy.

The point is, hierophant = sec taser dodging practice.

**Comment by WarRage333** [comment]
Dumb nerf just for the sake of nerfs

**Comment by ExcessiveUseOfCobblestone** [comment]
@WarRage333 n-n-n-no?

They were "nerfed" so each roundstart pack is viable in their own way (minus res, I know).

**Comment by Arianya** [comment]
I agree with removing the advanced scanner, but crusher kit *needs* the webbing to be able to carry basically anything in addition to it's mining tool. It's already a reduced inventory compared to the KA, on top of being a melee weapon in an area that it's better to be ranged in.

No one's going to take the crusher kit if it means they can't carry the bare necessities like their shelter capsule/survival medipen, let alone secondary stuff in the kit like the extinguisher.

**Comment by ExcessiveUseOfCobblestone** [comment]
You put it in your backpack or mine until you get the necessary items for webbing.

**Comment by Arianya** [comment]
@ExcessiveUseOfCobblestone you do realize the crusher only fits in the back slot
and requires two hands to wield at all times

It's not even like people were taking crusher kit for the webbing, it was for the advanced scanner. I agree with dialing back some of the powercreep on the kits but the webbing is an integral part of using a crusher (and has been in that kit as long as it has existed, AFAIK), all you're going to do is have people never take the crusher kit because its easier to just buy crusher + webbing with some points then try to use the crusher without webbing.

**Comment by ExcessiveUseOfCobblestone** [comment]
I can't imagine that it's easier to buy 2 items than it is to pick your favorite, mine, then buy the other and use the remainder for misc items. Now one choice may be preferred over the other, but it's nbd considering the Crusher is hard-mode so it's not meant to have equalized purchasing.

Yes I realize it's 2H and fits in the back slot, that's why you pull either the crusher or the backpack, and that's why I have no intention of including 7 extra slots to store things in for free along with it.

**Comment by Shadowflame909** [comment]
Man. Meleeing megafauna like bubblegum and surviving is already hard mode. Why ruin a gimmick. It's not like it particulary has Any advantage over a dual ka and a backpack.

**Comment by SgtxRootbeer** [comment]
@ExcessiveUseOfCobblestone 
You can't pull a backpack if you're pulling a crate, unless you want less minerals to get to the station 
:Bigfuckingconspiracythonk:

**Comment by ExcessiveUseOfCobblestone** [comment]
@SgtxRootbeer t-t-then use the mining bags (hint: people with the boxes only take 1 so there's freebies in the lockers), get enough for webbing (cheap), then forgo your backpack and switch to crate? You just can't carry a bunch of epic loot roundstart. Reminder this is ROUNDSTART items, not your entire kit for the round.

@Shadowflame909 you can carry survival pens, hivelord cores, etc. in your backpack. To access them you'd have to put away the crusher anyways, whether you're pulling them from webbing or backpack. Nothing changes except where you're clicking to pick it back up in that regard.

How is the gimmick taken away if you have to pull the crusher vs putting it in your backpack? Bad faith arguing...

Also > roundstart crusher bosses

That's your fault, not going to spoonfeed you items because you don't want to actually mine as the miner.

@Arianya All you need is analyzer and a light to mine. Your internals can fit in suit storage. You can mine > get enough points for webby > epic boss kill to your hearts content. Again, literally the point of this PR is to pull some stuff out of roundstart so you have to work for it.

It's currently fine as is. These are roundstart items, you can mine to fill in the gaps in your kit. If it ends up being a big deal I can edit it to reflect in-practice problems rather than poor theorycrafting because you picked the challenge item and got a challenge. I am not changing it unless a maintainer says to.

**Comment by Hatfish** [comment]
why did you remove webbing from the crusher, it's there explicitly because your back slot is eaten by your weapon already.  this just makes using one inventory management hell.

**Comment by Shadowflame909** [comment]
Yeah. Who in there right mind pulls there webbing away. When they heal they put it on there back. Dragging it around would slow you down. Make it more easy to dip into lava and lose your only source of light... that's another thing. How will miners see! They only have two pockets cobby. They either take no satchel, no flash light or no scanner. Just give them there webbing back. The gimmick turns from a challenge to insanity. They won't function.

**Comment by SgtxRootbeer** [comment]
@ExcessiveUseOfCobblestone 
The reason it only fits on the back is because it's like a fireaxe in terms of damage except it's easily aquired at round-start so it needs a Nerf.

To that end, like you've been saying, anyone can buy a webbing. That means a miner with a KA can still get more total space than a KC.

**Comment by ExcessiveUseOfCobblestone** [comment]
Comparing against the KA which the crusher is explicitly meant to be more difficult than is bad faith arguing.

Again, 1 Bag, Few more in your backpack, 1 Light, 1 Analyzer in backpack. Drop crusher and pick up as needed. Not difficult more than posters are making it.

**Comment by SgtxRootbeer** [comment]
Yes, the crusher is harder to work with, and good job, you found the 1 way to use the crusher without a webbing. You took the mining voucher, a tool to encourage variety and give players different starting options and found a way to shoe-horn and minimize the possible options that players had.
Previously players could use webbing and stuff bags into it, or a single bag plus a crate, and stuff a KA or extra shelter capsules into it, or maybe some first aid kits and tools or something else. 
But congratulations, you found a way to ensure every player with the KC has either the meta loadout, or they're gimping their mineral output.

**Comment by Shadowflame909** [comment]
There are two traits to being a great coder cobby. 1. Is knowing how to code. 2. Is listening to criticism and feedback. The maintainers ultimately give it, but it doesn't hurt to listen to what the general public has to say either. There's literally another PR doing exactly what you did, but with a slight change that everyone disagreed with you about.
If there's one thing I want you to take away from all this backlash it's that there's more to a conversation then thinking about a way to counter what someone said beforehand. Just take feedback...

**Comment by Repukan** [comment]
Someone not wanting to listen to your dogshit godawful opinions =/= Not taking and accepting criticism

**Comment by ExcessiveUseOfCobblestone** [comment]
> if I don’t agree with your argument I’m a bad coder

I take criticism all the time, I just don’t agree with the argument and find a minor change that can be told by a maintainer to compete with this pr is absurd. We joke about people writing 50 pages for balance changes, now we’ve gone to making different competing prs for each minor detail.

@SgtxRootbeer it’s a starting kit not your kit for the entire round unless you choose to. 

I’m just going to stop posting and let maintainers decide if I need to change my pr or not. Everyone, myself included, is not budging on the issue.

**Comment by Shadowflame909** [comment]
Didn't baiting people and being a general shitter get you permabanned? It's the opinion of everyone who likes the PR but disagrees. We disagree with this one item removal that cobby won't budge on. Unless a maintainer forces them too.

**Comment by BeeSting12** [comment]
This PR is better than @Arianya 's PR

**Comment by ExcessiveUseOfCobblestone** [comment]
> literally nolifed shaft miner
> the doesn't play argument

When will people learn :(

**Comment by iskyp** [comment]
>the webbing is a necessity

HAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHAHA

**Comment by davethwave** [comment]
Honestly the advanced resonator should probably have a 3 by 3 blast zone instead to make up for it's inequalities, mostly not being a robust weapon compared to everything else miners get.

**Comment by Shadowflame909** [comment]
@iskyp  I'm not wrong though. If your forcing people to take confusing steps to get something remotely close to there old set-up. It's the same as a removal. I'd take the drone PR as an example. If you make things extremely troublesome for people, they're just gonna avoid the trouble altogether. 

This will destroy the crusher voucher as an option. I hope the maintainers agree with my sentiment. Cobby says so themselves..."It's meant to be more difficult" I take the belief that making things harder and less fun for difficulty's sake isn't really a good idea.  People want a challenge! Not to fall into a round start noob-trap that they have to work themselves out of. They'll just choose the other kits and buy the items separately...completely the same as removing that kit anyways.

**Comment by zaracka** [comment]
I tested the whole webbing/no webbing thing. You can use it without the webbing, but it's kind of like ultra hard mode instead of just plain hard mode. Definitely doable, but unfair to the average or new player.

Basically, you have to decide on whether to bring a fire extinguisher or a shelter capsule. You don't have any extra room for miscellaneous items that can fit in a webbing either. Not sure, but this might include legion cores.

**Comment by zaracka** [comment]
Now if the webbings are to be removed... Can you do the following?

- Add a fire extinguisher to the Survival Capsule and Explorer's Webbing kit.

- Take the shelter capsule out of the belt and add it to the kit.

- Make it so there's a chance you can get the alternate webbing.

**Comment by Repukan** [comment]
There's fire extinguishers in the base, you can get the free capsules from the lockers.

**Comment by zaracka** [comment]
Emergency toolboxes are for emergencies, not mining.

**Comment by davethwave** [comment]
I agree that you shouldn't start with capsules since their only really useful later on when you get deep into lavaland.

**Comment by Shadowflame909** [comment]
What do you mean by later...The second time you mine?

**Comment by zaracka** [comment]
Here's some reference. There are 3 mining lockers on the main station and 3 in the mining station, each has a shelter capsule inside. The webbing kit has one in the webbing. You can buy more or the luxury version from vendors.

**Comment by ExcessiveUseOfCobblestone** [comment]
On one hand I think the capsule gives fake value to the quite niche storage item.

On the other hand, as you can imagine, I don't really want to continue playing balance merry-go-round. I'd do those changes after viewing stats post-this just to confirm that it should be given more bones (which I personally don't mind)

**Comment by Shadowflame909** [comment]
Honestly, does anyone other then cobby think a change like this is needed? Or is this just a nerf because something could be nerfed.

**Comment by zaracka** [comment]
Mining is due for some nerfs to make it more challenging in my opinion. This PR, specifically, makes sense because one kit is obviously better than the others instead of being personalized or situational.

**Comment by Shadowflame909** [comment]
@zaracka I'm gonna try to keep this one short and concise...

1. I believe this PR doesn't just equalizes Roundstart Mining Options, it actually slows all of the options down in general. 

1a. It removes advanced scanners, thus making the first batch of mining less efficient. Which means that more time is wasted having the ores appear on the scanner. It also encourages you to buy the advanced scanners, which means less points on say upgrades or survival pens. I have a fear that this will work alongside the impending RND nerf. Which will double-down and make it even worse then expected. It'll probably delay tier 3 upgrades by at least 15 minutes more. I'm not sure if that seems like a lot to you. But with the shortness of our rounds, it's pretty impactful to me.

2. This is not the superior PR, the other one is because whilst removing advanced scanners and making mining slower. By removing the webbing for the crushers, it also makes the QUANTITY of the ores you can obtain slower. Which I don't think the goal of this PR was to make the station receive fewer ores in general. Thus helping prove point 1a, if mining is forced to deliver fewer ores then it's actually a general RND + Mining nerf.

3. Lastly, I feel like with how easy it is to die with a kinetic crusher. There's always a chance that they won't be coming back with ores anyways. By removing advanced scanners, there's an increased chance of ALL miners in general falling ill to ash-storms. Which you may say "Git good and that won't happen" I'm saying that this will have unforeseen consequences on mining in general. At least for a week after this PR gets merged.

You may say this is all based on hysteria and hyperbole. Yet, it's true, isn't it? You work faster with the better equipment. So wouldn't you go slower by that much with barely any equipment? So, I doubt making it so lava land miners can't mine wasn't the intention. Also with the info, the title gives us, nerfing the speed of mining shouldn't be the intention either. It should be making no kit better than the other. So is this title misdirection in the most classic sense?

**Comment by zaracka** [comment]
Right, advanced scanners are so impactful that the other kits are seemingly useless. I agree, the crusher kit needs webbing. The crusher kit is supposedly the hard mode kit, you don't need to take it. Advanced scanners do not have an effect on dying to ash storms.

Here's what I was able to mine (I've done better, but this run was for testing, so I kind of rushed it):
used the kinetic crusher
ORM was not upgraded (default machine parts)
just my ores, not the other miners who dropped them after me
I had an advanced scanner
I made 4872 points
8 minutes from being spawned in

ores

I could have mined more diamonds (I've gotten 20 with these exact same factors before, but I didn't kill many watchers during this run). This was all from one trip, my first trip.

**Comment by SpaceManiac** [comment]
Without the advanced scanners, *are any of these packs useful to non-experts*?

**Comment by Shadowflame909** [comment]
I guess I'm not as good as mining as I thought. I never mine that much without at least some upgrades. I guess fauna have something to do with that though. I honestly want to see what pro strategies you used for that. Because the average miner would only get 1/3rd of that.

**Comment by 81Denton** [comment]
>Without the advanced scanners, are any of these packs useful to non-experts?

The fulton kit and maybe the webbing.

Minebot is only good as an ore box until you get the sentience upgrade. Crusher without a webbing/survival pen is pointless for people who don't know how to abuse the AI. The webbing kit is well, just a webbing and a capsule you can get for free from lockers. Resonator is easily the worst, it's bad for both mining and fighting.

**Comment by ExcessiveUseOfCobblestone** [comment]
Apart from the range differences is there something else the advance scanner does that i'm not aware of? It's just a range upgrade to my knowledge.

I can't see how not having it somehow makes mining that bad outside of the obvious "you can no longer see as much ore as you used to".

@Shadowflame909 minerals are coded to spawn in "veins" so if you just go to one mineral you'll likely fine more, meaning the hit on mining is pretty minimal but it DOES give you something to progress in terms of mining benefits. Sure it's not as powerful as the full screen one but that's the point lol. I've already said

>If the lesser one is so trash then a separate PR can be made to remove the lesser and make the advanced be the default.

 in the first post, not sure besides your 'us v evil coder' mentality would you think I'm nerfing to nerf. Just trying to make each pack viable even if the approach indirectly added a layer of seemingly-arbitrary progression not currently present (which is also found in the other PR).

**Comment by Shadowflame909** [comment]
Honestly. I don't think, I can really agree with this PR and that's fine. So, It's not very healthy for me to continue trying to keep arguing pointlessly. I've made my stance against this very clear and my Favor for this not existing/ The one with the webbing to be merged instead. So what I'm trying to say here is, I think I'll just agree to disagree.

**Comment by ExcessiveUseOfCobblestone** [comment]
Arguing the PR isn't an appropriate solution vs my motivations for making the change are two different conversations with you plaguing my PR with the latter under the veil of having the first.

I'm perfectly fine with you disagreeing with the solution or even the approach, I'm not ok with you implying malicious intent based on the approach I chose.

**Comment by Shadowflame909** [comment]
I apologize for making such assumptions. I'd like to detract any sense of me portraying you out as a super-evil.

**Comment by zaracka** [comment]
Most people don't bother looking at what kits are available because they just want the advanced scanner. There might be problems with the other kits, but we'll never know unless people use them.<br/><br/>

Mine every kind of ore, not everything you run into (avoid mining iron just because it's there). Focus on ores near the outside of rocks, so that you're not wasting time mining to them. If you can't find diamonds, butcher some watchers. You don't need to kill fauna, you can just walk passed them. Fighting megafauna is a choice.<br/><br/>

Here's a run I did with the default scanner, no kit, and the accelerator. Same factors, the ORM was not upgraded, just my ores, in roughly 8 minutes.<br/><br/>

ores2

**Comment by SpaceManiac** [comment]
As a player I will miss the advanced scanner, the only item in any of the packs I actually used aside from the webbing when I actually manage to get a drill or sonic jackhammer. From a design perspective, giving the crusher without a webbing seems a little odd, given how annoying it is to carry otherwise.

[Statbus for May 2018](https://atlantaned.space/statbus/stat.php?stat=mining_voucher_redeemed&year=2018&month=5) (March and April are very similar, the packs were [last changed mid-February](https://github.com/tgstation/tgstation/pull/35633)):

1. Crusher Kit: 	1318
2. Resonator Kit: 	933
3. Survival Capsule and Explorer's Webbing: 	637
4. Extraction and Rescue Kit: 	259
5. Mining Conscription Kit: 	85
6. Minebot Kit: 	67

The crusher kit is on top followed closely by the resonator kit. The webbing kit is still extremely popular despite basically being a noob trap with how little utility it provides. The minebot and conscription kit battle it out for least popular, one being a near-useless animal and the other being pretty much just a traitorbro kit.

That said, it seems theorycrafting is getting us nowhere fast. Unless anyone comes forth with an actual *design* for these packs, I'm thinking either merge or testmerge this and see what players think, or just closing both PRs until someone feels up to rebalancing kits holistically rather than piecemeal as we have been doing.

**Comment by zaracka** [comment]
I think the logic for most miners is more ores = more points, I can use said points to buy everything from the other kits. So it's logically the best choice, even though the impact isn't actually _that_ big, at least short-term.

It definitely leaves a bad taste in my mouth when I see people taking the crusher or resonator kit and leaving everything behind except the scanner. In my opinion, the more personalized or situational the kits the better. I'd like to see different miners with different "setups" instead of everyone taking what is "meta." It doesn't help that mining equipment is balanced around Lavaland, so it becomes incredibly unbalanced when used on the station. Plasma cutters are also ridiculously better to the point where miners don't bother mining until they're researched.

This might be a bit anecdotal, but I had a miner legitimately get mad because I didn't research advanced cutters, even though I had researched normal cutters. The miner then ordered a mining borg to research it, which ended up using 10, 000 sci points. Looking at the code, the difference between the two doesn't seem extreme. This was close to round-start too.

A lot of the time miners are given everything they ask for, but fail to deliver many ores because they're just playing for Lavaland loot. Making points more impactful and harder to get might counteract this. Survival medipens are only 500 points. The brute kit, which is supposedly worse, is 600 points. I'm thinking about how the supposedly useless equipment like diamond pickaxes could be useful if xeno golems were sent to mine.

**Comment by Shadowflame909** [comment]
>'m thinking either merge or testmerge this and see what players think, or just closing both PRs until someone feels up to rebalancing kits holistically rather than piecemeal as we have been doing.

I'm not sure why you'd want to testmerge it. The players are going to see a pure nerf PR as a negative thing. It's not like current mining is gamebreakingly easy.

**Comment by ExcessiveUseOfCobblestone** [comment]
The resonator is getting buffed, i'm waiting for the other resonator buff PR to close/merge before I toss my hand in there though.

If I did a major rebalance to the starting equipment in one fatal swoop I'd get meme'd on for not atomizing the changes. I can't even remove 2 items without an 80+ comment PR.

@Shadowflame909 Your feeling of "I must get more minerals than the pros" is nothing I can change. If you feel the need to powergame that's entirely player-interactions or self, neither of which are my problem for the purpose of the PR. There's not exactly a pro-strat to mining-proper either, most "pro" miners are "pros" because they can kill megafauna.

@SpaceManiac I was under the impression it's **suppose** to be cumbersome, it doesn't make sense to me design-wise to make a roundstart item that takes the backpack slot then give them a backpack slot but on their belt. 

Before I get the gotcha of "haha it doesn't actually carry literally everything", it doesn't matter when it carries all the items practical to a miner.

**Comment by Shadowflame909** [comment]
@ExcessiveUseOfCobblestone 
I'm glad the resonator won't be just pickaxe with a different icon but you missed my point entirely. It's just been proven that people who are great at mining will still be able to mine the same amount round start. So isn't this PR just hurting everyone else? It's cumbersome because you can take fewer items then you could take with a backpack, and you're forced to put your mining satchel in your pocket. 

Thus losing one of your pockets. You can only take 5 items compared to 30. That's pretty cumbersome to me cobby. Also, since you wouldn't be able to drag an ore-box. By the time you'd be able to get the webbing you already have a bag of holdings that you can put your crusher in. So why make the webbing useless?

**Comment by ExcessiveUseOfCobblestone** [comment]
@Shadowflame909 Sure it'll be more difficult, that's the point of the PR. The point of contention though is "is that a problem"? If you're good at mining you're good at mining, if you're not so good then you're not so good _but you can be if you choose_. I don't think ensuring an equal amount of minerals gets passed every round is a good thing. Also I'd disagree on the use of "powergaming" as a word to describe being able to do the job faster/better than someone else.

We're never going to agree on the whole "the crusher is suppose to be cumbersome" point because we actually both agree on it, just disagree to what extent it should be cumbersome. The whole "boh" thing is silly because it's just ~500 points for the webbing which is typically more than you can get just by going outside the base, way before BoHs are even thought of (especially after Science loses the pseudo-monopoly thanks to the Silo). You being forced into a decision of going back/ trekking on possibly going less prepared than you like to me is interesting, and actually the whole concept behind games like Oregon Trail/FTL/etc. Having to constantly make tough decisions is what I personally want on a game advertising itself as a "paranoia-laden metal death trap", but that's a discussion for somewhere else.

**Comment by zaracka** [comment]
@Shadowflame909 I don't fight megafauna, so I'm not sure why you think I'm a "pro miner." Have you killed any megafauna? If you can kill megafauna but somehow have an ore problem, maybe the reality isn't the advanced scanner, but lack of motivation to mine in general. Are we sure this isn't the min-max miner mentality plaguing SS13?

People are taking kits for the scanner only. If it's _that_ good, then maybe make the default scanner the advanced scanner. I don't think this is the case though, it's just the powergame mentality most miners have nowadays. "I need survival pens because Lavaland is harsh." _fills bag of holding and webbing with survival pens to murderbone or validhunt_ Definitely clashes with typical chemists who rarely make medicine. Maybe regular fauna need to have less health but be more plentiful? Why haven't any miners proposed that?

**Comment by SpaceManiac** [comment]
>If I did a major rebalance to the starting equipment in one fatal swoop I'd get meme'd on for not atomizing the changes. I can't even remove 2 items without an 80+ comment PR.

You might get memed on, but it wouldn't be a legitimate complaint. Atomization should be invoked when a PR bundles multiple *unrelated* changes. I can't promise there won't be an enormous discussion though. That said, I would advise against a rebalance. What these kits really need is a *redesign*. The goal should not be putting each kit at equal power, but making the idea of roundstart mining kits actually make sense, as kits with interesting variations, rather than a set of shortcuts to your favorite equipment.

I'm not a typical enough miner to truly weigh in on the effect this will have on mineral returns to the station. I don't fight megafauna. I strictly mine minerals because I remember how annoyed I am when I play engineer or scientist and there's no minerals. My typical death is by failing to medipen myself during a goliath fight or by sonic jackhammering gibtonite. 

---

Here's another data point: number of vendor points which each kit corresponds to. Keep in mind that 1000 points at no upgrades is 20 diamond/titanium ores, 33 uranium ores, or 66 plasma ores, and that zaracka was able to get 4000 points in about 8 minutes of mining.

Before this PR:

* Shelter Capsule 400 + Webbing 500 = **900**
* Webbing 500 + Scanner 800 + Resonator 800 = **2100** (bonus mini extinguisher)
* Webbing 500 + Scanner 800 + Crusher 750 = **2050** (bonus mini extinguisher)
* Fulton Pack 1000 + Fulton Beacon 400 + Thirty Marker Beacons 300 = **1700**
* Minebot 800 + KA Passthrough Mod 100 = **900** (bonus welding tool and helmet)
* Mining Conscription Kit 1000 = **1000**

After this PR:

* Resonator 800 = **800** (bonus mini extinguisher)
* Crusher 750 = **750** (bonus mini extinguisher)

So this knocks the two best packs down to being the worst. That'll shake things up for sure.

**Comment by zaracka** [comment]
I mined with the mentality that all departments should get at least 5 sheets of all ores. I take into account that R&D can make bluespace crystals with plasma and diamonds. When I'm an engineer or scientist and there's 10 diamonds in the ORM, I take 2. I take 5 if there's more than 20. I take 1 if there's 5 or less. You'll see scientists taking all the ores and not using the majority of them. I've upgraded the entire station with tier 4 parts with 5 of each ore, 5 glass, 10 metal sheets.

I did a run where I didn't really care about the station and made roughly 2000 points even though the ORM was upgraded with super matter bins, so I think tier 2 parts. I was able to buy enough pens to kill the hierophant and I had fun, but I'm sure the station wasn't doing as good as it could have. I remember a round where my mining allowed the station to have 4+ mechs. Of course the crew acted pretty crappy and half of them ended up being destroyed despite the round being extended.

**Comment by Shadowflame909** [comment]
@zaracka I can kill every megafauna except for the colossus.

@ExcessiveUseOfCobblestone I don't know how you can expect people to be happy with a straight up nerf and accept it with wide open arms. I mean, as long as the webbing stays. I'd probably cease my aggressive commenting anyways. It's just, why would you force people to start off with the crusher looking like they just went on a cliche shopping spree in a clothing store and are struggling to carry everything they can.

I mean, the webbing would be less important if we could carry more then 50 units of ore at roundstart. But being forced to pull your backpack to carry all that you need is absurd.

**Comment by zaracka** [comment]
Do you use ore boxes? Those are available roundstart. Applied Bluespace which lets you carry virtually unlimited ores is an early tech and coincides with autocloning, honestly more useful than cutters.

**Comment by Hatfish** [comment]
i just want the alternate webbing to be reliably available because the default one you get from the vendor looks like garbage compared to it, everything else is just peanuts.

**Comment by Shadowflame909** [comment]
@zaracka I think you misread what I said and goofed.

If this PR gets merged you can no longer drag an ore box with the kinetic crusher as you're forced to drag a backpack instead to have a chance at surviving.

**Comment by zaracka** [comment]
SpaceManiac is right. This whole thing needs redesign. I'm hoping whatever happens works toward deterring powergaming though. The reality is there's miners who will mine so much you never run out and miners who will mine so little it's as if they haven't mined at all.

Your logic is that they'll mine less because of lesser capability (worse equipment), my logic is that because they have worse equipment, they're incentivized to get better equipment (with points) and thus forced to mine ores. Think about game design, it doesn't make sense to start with the best. That was why the original PR was closed.

You should not be dragging a backpack, that's honestly really weird and crazy. The real argument is you can't take more equipment with you during your first trip, like fire extinguishers and shelter capsules. You only have 2 pockets after all. I didn't forget about survival pens, I put those in my boots, but we can't expect everyone to know you can do that.

**Comment by iskyp** [comment]
guiness world record for longest argument about menial shit given to -tg-station13 github after 3 year long argument about a single item takes place

**Review by SpaceManiac — APPROVED** [review]
Probably going to merge this. The kits will need further work later.

**Comment by Shadowflame909** [comment]
What do you think about the alternative PR version of this, the one where the webbing stays and doesn't force you to not drag your backpack instead of an orebox at roundstart?

**Comment by ExcessiveUseOfCobblestone** [comment]
You can have both a backpack and ore box, these aren't mutually exclusive items. You will just have to do slightly more inventory tetris by temporarily dropping the Crusher, which shouldn't be a problem for someone who I believe has played before a lot of our inventory tetris QoLs were added.

I agree @SpaceManiac with the mining rework, I just have to throw a bunch of excuses so I don't have to do it :P 

Need to play mining a bit after this (should it get merged) and see the actual effects to gauge a more proper solution. I'm leaning to just removing KA outright and flesh out resonator so roundstart packs should they even remain be based around various playstyles of a singular item.

**Comment by Shadowflame909** [comment]
Yikes. What a sad day it is for ores. Well, I tried. GG

**Comment by ExcessiveUseOfCobblestone** [comment]
lol I'm reworking roundstart mining entirely after I see how these play out (which I can't do since I'm on leave until ~27th)

**Comment by KorPhaeron** [comment]
obviously I ignored this for 3 weeks because I hate mining package PRs but why did you feel the compulsion to make them all equally taken anyway

**Pull request merged**
